shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -1183,25 +1184,22 @@ \subsection{Reward Update Calculation}
 
     (RA) edge (C);
 
-    \draw[->, implied, thick]
+    \draw[->, implied, ultra thick]
     (FR) edge (T)
     (FR) edge (RA)
 
-    (R) edge (FR)
     (R) edge (T)
     (R) edge (RA);
 
-    \node (TP) at (3, 1.25) {Total pot};
+    \node (TP) at (3, 1.15) {\LARGE Reward pot};
 
-    \draw[->, toTotPot, thick]
+    \draw[->, toTotPot, ultra thick]
     (FR) edge (TP)
     (R) edge (TP)
 
     (TP) edge (RA)
-    (TP) edge (FR)
     (TP) edge (T);
-
-  \end{tikzpicture}
+    \end{tikzpicture}
```

Comment:
why?

Reply:
There were two things about it that were different than the matching `\begin`. They were not vertically aligned, and there was no blank line after the `\begin`.